### DISCUSSION TITLE
How to run multiple servers

### DISCUSSION THREAD

**Discussion by psynezo** [Minestom/Minestom#1768] [Help] [answered]
The question is in the title. I need to run multiple servers in one application, is there a solution?

**Comment by BasToTheMax** [comment]
Use instances

**Reply by psynezo** [reply]
Okay, how?
Send an example and I'll close the question.

**Comment by DevBlokJochem** [comment, marked as answer]
At the bottom of this article you can find an example how to create an instance.
https://wiki.minestom.net/world/instances
You can set the chunkloader to an AnvilLoader with a path. This path should be a location to the world.

(instanceContainer should be instance)

**Reply by psynezo** [reply]
For example, is it okay to have more than 30 instances at the same time and have 10 or more players on them.
And yet, is it possible to set different events between 30 instances.

**Reply by DevBlokJochem** [reply]
Yes, use eventnodes and make a condition for each event node that they have to be in a specific instance.

**Comment by Tofaa2** [comment]
Are you looking to run multiple server instances or normal instances (worlds)? Normal Instances should be fairly simple its just creating a new one. Server instances you need a backbone with kubernetes or something else

**Reply by psynezo** [reply]
In general, it depends on how many maximum instances of worlds can be used at the moment and whether events can be divided between them, for example.

**Reply by DevBlokJochem** [reply]
Unlimited till your cpu/ram says goodbye
